tests/planner/test_World.cpp

```diff
@@ -0,0 +1,93 @@
+#include <dart/dart.hpp>
+#include <gtest/gtest.h>
+#include <aikido/planner/World.hpp>
+
+using std::shared_ptr;
+using std::make_shared;
+
+class WorldTest : public ::testing::Test
+{
+public:
+  using SkeletonPtr = dart::dynamics::SkeletonPtr;
+
+  WorldTest()
+    : skel1{dart::dynamics::Skeleton::create("skel1")}
+    , skel2{dart::dynamics::Skeleton::create("skel2")}
+    , skel3{dart::dynamics::Skeleton::create("skel3")}
+  {
+    mWorld = std::make_shared<aikido::planner::World>("test");
```

Comment:
Should `World` have a similar `create()` helper to the above one that `Skeleton` has?